### ISSUE TITLE
Example virtual secondary configuration raises "Value is not convertible to bool." error.

### ISSUE THREAD

**Issue by rstanchak** [advancedtelematic/aktualizr#1504]
I faced an issue with the instructions for "Simulate a device without building a disk image"
https://docs.ota.here.com/getstarted/dev/simulate-device-workstation.html which also appears to be present in the provided example configs.

The VirtualSecondaryConfig constructor is expecting a boolean field for the "partial_verifying" key

https://github.com/advancedtelematic/aktualizr/blob/2403384097448042c825c485e905deb130ce9fcc/src/virtual_secondary/virtualsecondary.cc#L15

However, in the example configs, the field is a string
https://github.com/advancedtelematic/aktualizr/blob/2403384097448042c825c485e905deb130ce9fcc/config/secondary/virtualsec.json#L4

https://github.com/advancedtelematic/aktualizr/blob/2403384097448042c825c485e905deb130ce9fcc//docs/ota-client-guide/modules/ROOT/examples/virtualsec.json#L4

As a result, running the 'quick start' with virtual secondaries fails with a cryptic "Value is not convertible to bool" error:

```
user@host $ aktualizr -c .
Aktualizr version 2019.11-0-g6633d0d0 starting
Aktualizr is currently running as non-root and may not work as expected! Aktualizr should be run as root for proper functionality.

Reading config: "./sota-local.toml"
Current directory: /home/roman/steer/task-ota/aktualizr-virtual-devices/primary0
Use existing SQL storage: "storage/sql.db"
Could not create "/var/run/aktualizr-session" securely, reboot detection support disabled
Flushing report queue
Value is not convertible to bool.
```

**Comment by patrickvacek** [comment]
Hello @rstanchak, sorry that you ran into this. I'm trying to reproduce locally and I've run into a similar error. This is strange, as that code hasn't changed for a while and I've tested it quite recently. We're still looking into it. In the meantime, you can try leaving out that line, as the default is `false` anyway.

**Comment by patrickvacek** [comment]
Thanks again @rstanchak for reporting this. We've now sorted out what happened here. We recently bumped the version of our json parser and managed to overlook that the boolean parsing changed. We have a fix available in #1505, which is basically just to remove the quote marks around `false`. We'll merge that soon and start working soon on a new release with that fix.
